scripts/create_test_env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
can we remove the scripts since all the test setup will be specified in the travis.yml file?

Reply:
removed